ansible/roles/test/files/ptftests/py3/advanced-reboot.py

```diff
@@ -1457,14 +1467,24 @@ def wait_until_teamd_goes_down(self):
 
         while teamd_state == 'active':
             time.sleep(1)
-            dut_datetime_during_shutdown = self.get_now_time()
+            try:
```

Comment:
would be better to define a new function - run_and_ignore_failure, as you use it more than once